architecture/index.adoc

```diff
@@ -0,0 +1,71 @@
+[id="overview-of-architecture"]
+= Overview of architecture
+include::modules/common-attributes.adoc[]
+:context: overview-of-architecture
+
+toc::[]
+
+{product-title} is a cloud-based Kubernetes container platform.
+The foundation of {product-title} is based on Kubernetes and therefore shares the same technology.
+To learn more about {product-title} and Kubernetes, refer to xref:../architecture/architecture.adoc#architecture[product architecture].
+
+[id="about-installation-and-updates"]
+== About installation and updates
+
+As a cluster administrator, you can use the {product-title} xref:../architecture/architecture-installation.adoc#architecture-installation[installation program] to install to deploy a cluster on:
+
+** Installer-provisioned infrastructure clusters (IPI)
+** User-provisioned infrastructure clusters (UPI)
+
+[id="About-control-planes"]
+== About control planes
+
+The xref:../architecture/control-plane.adoc#control-plane[control plane] manages the worker nodes and the pods in your cluster. You can configure nodes with the use of machine config pools (MCP).
+Machine config pools are groups of machines such as, control plane components or user workloads. These groups are based on the resources handled.
+{product-title} assigns hosts different roles. These roles define the function of a machine in a cluster.
+The cluster contains definitions for the standard master and worker role types.
+
+You can use operators to package, deploy, and manage services on the control plane. With operators, you can use `kubectl` and `oc` commands.
+Operators are important components in {product-title} because they:
+
+* Perform health checks.
+* Enable the way to watch an application.
+* Manage over-the-air updates.
+* Ensure applications stay in the specified state.
+
+[id="how-to-develop-your-containerized-application"]
+== How to develop your containerized application
```

Comment:
This heading sounds like it would be a procedure. You might consider something more conceptual, such as "Containerized applications for developers"